### PR TITLE
[msbuild] Avoid Dictionary clashes in IBToolTask's mapping dictionary

### PR DESCRIPTION
**Pull request by jstedfast**
Fixes https://bugzilla.xamarin.com/show_bug.cgi?id=54914

### CODE REVIEW

File: msbuild/tests/Xamarin.iOS.Tasks.Tests/TaskTests/IBToolTaskTests.cs [hunk 1]
```diff
@@ -169,5 +169,76 @@ public void TestAdvancedIBToolFunctionality ()
...
+		[Test]
+		public void TestGenericAndDeviceSpecificXibsGenericFirst ()
+		{
+			TestGenericAndDeviceSpecificXibsGeneric ("View.xib", "View~ipad.xib");
+		}
+
+		[Test]
+		public void TestGenericAndDeviceSpecificXibsGenericLast ()
+		{
+            TestGenericAndDeviceSpecificXibsGeneric ("View~ipad.xib", "View.xib");
```
Comment (chamons) [inline msbuild/tests/Xamarin.iOS.Tasks.Tests/TaskTests/IBToolTaskTests.cs]: Spaces instead of tabs?

Comment (jstedfast) [inline msbuild/tests/Xamarin.iOS.Tasks.Tests/TaskTests/IBToolTaskTests.cs]: yea, looks that way

**Comment by jstedfast** [comment]
ping

**Comment by jstedfast** [comment]
Yep, it is :)

**Pull request merged**